src/main/resources/dbchangelog.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                    xmlns:pro="http://www.liquibase.org/xml/ns/pro"
+                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <changeSet  id="1"  author="dgwiazda">
```

Comment:
Ta tabelka jest na razie taką wstępną, potem sobie to zmienimy, ale jeśli chodzi o id to może przyjmimy sobie taką konwencję: 
<unix_timestamp>_<jakas_znaczaca_nazwa_tego_co_jest_w_changesecie>